### PR TITLE
feat: add initial reviewpad configuration

### LINKED ISSUE #223: [Feat]: reviewpad
### Feature request

add reviewpad
https://reviewpad.com/

### PR DESCRIPTION
**Pull request by schmelto**
Fixes #223

### CODE REVIEW

File: reviewpad.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    color: 'E50009'
+    description: 'Denotes a change 1000+ lines'
+
+workflows:
+  - name: label-pull-request-with-size
+    if:
+      - rule: $size() <= 9
+        extra-actions:
+          - $addLabel("size/XS")
+      - rule: $size() => 10 && $size() <= 29
```
Comment (ferreiratiago) [inline reviewpad.yml:29]: This rule has a parsing error. The sign `=>` should be `>=`. Like you have it [here](https://github.com/schmelto/Portfolio/pull/224/files#diff-d7a202cc076357dca8f9a8931b252f8ba4803ea0d12b140e62bea228410ad634R41). The same applies to the conditions bellow 😄

File: reviewpad.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        extra-actions:
+          - $addLabel("size/XL")
+      - rule: $size() >= 1000
+        extra-actions:
+          - $addLabel("size/XXL")
+
+  - name: empty-description
+    on:
+      - 'issue'
+      - 'pull_request'
```
Comment (ferreiratiago) [inline reviewpad.yml:48]: This configuration does not happen at the reviewpad configuration level but at the github action configuration level. More specifically [here](https://github.com/schmelto/Portfolio/blob/main/.github/workflows/reviewpad.yml#L4).

The same is true for line [56](https://github.com/schmelto/Portfolio/pull/224/files#diff-d7a202cc076357dca8f9a8931b252f8ba4803ea0d12b140e62bea228410ad634R56) and [65](https://github.com/schmelto/Portfolio/pull/224/files#diff-d7a202cc076357dca8f9a8931b252f8ba4803ea0d12b140e62bea228410ad634R65).

I understand why you did that. I guess that having this kind of configuration (where the rule should run, if `pull-request` or `issue`) is useful. I would create an issue for this on Reviewpad project. Please let us know what you think 😄

**Review by ferreiratiago — COMMENTED** [review]
Hi @schmelto Some comments regarding the reviewpad configuration :)

File: reviewpad.yml (hunk 1 above)
Comment (schmelto) [reply reviewpad.yml:29]: Thanks great spot

File: reviewpad.yml (hunk 2 above)
Comment (schmelto) [reply reviewpad.yml:48]: Yeah it would be nice!

Comment (ferreiratiago) [reply reviewpad.yml:48]: Hi @schmelto It turns out that you're 100% right. Totally miss that `on` feature 🤦‍♂️ . It works exactly has you have it in the pr. `issue` will run the workflow only on issues, `pull_request` only on pull requestes. Having both run for both issues and pull request. When no value is provided then it only runs for pull requests. Sorry for misguided you.

**Pull request merged**